libutils/json.h

```diff
@@ -416,6 +416,13 @@ void JsonArrayAppendObject(JsonElement *array, JsonElement *object);
   */
 void JsonArrayAppendElement(JsonElement *array, JsonElement *element);
 
+/**
+  * @brief Copy elements from JSON array `a` to JSON array `b`.
+  * @param a [in/out] The JSON array to copy elements to.
+  * @param b [in] The JSON array to copy elements from.
```

Comment:
They are both `in` parameters. But actually, `JsonElement *b` should be `const`. That will also make it clear.

Reply:
I guess `JsonElement *b` should no longer be `const` as it's freed ?